test/Microsoft.AspNet.Mvc.Xml.Test/XmlAssertTest.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Xunit;
+using Xunit.Sdk;
+
+namespace Microsoft.AspNet.Mvc.Xml
+{
+    public class XmlAssertTest
```

Comment:
add tests of
- differences in later text nodes e.g. `<a>hello<b>world</b>hello</a>` compared to `<a>hello<b>world</b>goodbye</a>`
- differences in later elements e.g. `<a><b>hello</b<b>world</b></a>` compared to `<a><b>hello</b<c>world</c></a>`, `<a><b>hello</b<b>goodbye</b></a>`, and `<a><b>hello</b<b attribute="value">world</b></a>`